assets/js/components/post-searcher.js

```diff
@@ -112,28 +115,42 @@ class PostSearcher extends Component {
 	}
 
 	onConfirm( selection ) {
-		this.setState( {
-			selection,
-		} );
+		const { results } = this.state;
+		let match;
+		// Check to that the selection is "valid".
+		if ( ! this.disallowedSelections.includes( selection ) ) {
+			match = find(
```

Comment:
Not sure why lodash was used for this before but we should be able to simply use `results.find` instead. We might only want to add a guard at the very top to return early `if ( ! Array.isArray( results ) )`.

Reply:
Updated